Move out VSCode plugin and composer packages

prabushi opened the pull request:
## Purpose
This PR removes the VSCode plugin and ast-model, bbe, distribution, documentation, font, lang-service, and theme composer packages from the ballerina-lang [master]. The implementation is moved to [plugin-vscode](https://github.com/ballerina-platform/plugin-vscode) repository.
Related PR: https://github.com/ballerina-platform/plugin-vscode/pull/1

## Approach
> Describe how you are implementing the solutions along with the design details.

## Samples
> Provide high-level details about the samples related to this feature.

## Remarks
> List any other known issues, related PRs, TODO items, or any other notes related to the PR.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

hevayo reviewed (changes requested):
It seems that we have removed doc-search package from composer which is used in API Docs

prabushi commented:
> It seems that we have removed doc-search package from composer which is used in API Docs
Updated the PR. The doc-search and syntax-highligher packages are kept in ballerina-lang repo. Only ast-model, bbe, distribution, documentation, font, lang-service and theme packages are moved to plugin-vscode repo.
